pkgs/by-name/fr/freeshow/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  pkgs,
+  appimageTools,
+}:
+
+let
+  pname = "freeshow";
+  version = "1.2.1";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.dmg";
+          hash = "sha256-OO0uQ6oS1GKiBGz3Qt9jDCY+qdWlgTOY+SiLJB5xQ3c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.AppImage";
+          hash = "sha256-RBI8IkxY6Xd36vCVDJy9sqpEisB/48hwQo+mg5XTCOs=";
+        };
+      }
+      .${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  appimageContents = appimageTools.extract { inherit pname version src; };
+
+  meta = {
```

Comment:
The meta parameters have [order](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes). You want `description` then `homepage`, and `mainProgram` then `platforms`.

Reply:
there's no enforced ordering in nixpkgs @SigmaSquadron

Reply:
I apologise, I assumed there was, as I was told to reorder attributes here: https://github.com/NixOS/nixpkgs/pull/324693#discussion_r1672827358.

Reply:
There's nothing in the manual nor that I saw in the contributing guidelines that enforces an ordering for `meta`, nor does CI enforce such an ordering. And merge conflicts should only really matter for treewides, not individual package changes.

Since there's no evidence to support an ordering, I would consider that an individual's opinion, not something we enforce in nixpkgs.